.cmakelintrc

```diff
@@ -0,0 +1 @@
+filter=-linelength,-convention/filename,-package/stdargs
```

Comment:
I think we should add a todo to enable eveything one day?

Reply:
I don't think so. We can talk about line length and fine-tune the autoformatter config to match this, but the others are disabled because our find modules are supposed to match the case of the cmake config modules shipped with distros. We need these find modules because some distros ship packages with missing/broken find modules (*cough* debian *cough*).

And stdargs one is disabled to support the media foundation module on macOS.